### ISSUE TITLE
Update helm-charts index for 1.7.0

### ISSUE THREAD

**Issue by bushong1** [aws-observability/helm-charts#46]
I'm trying to use the latest release via `https://aws-observability.github.io/helm-charts/index.yaml`, but it's still referencing v1.6.0 as the latest.

**Comment by samuelthan** [comment]
Likewise, seems like something broke in the release process for v1.7.0

**Comment by lisguo** [comment]
Thank you for bringing this to our attention. We have resolved the issue and v1.7.0 is available in helm:

```
helm search repo aws-observability
NAME                                             	CHART VERSION	APP VERSION	DESCRIPTION
aws-observability/amazon-cloudwatch-observability	1.7.0        	1.0.0      	A Helm chart for Amazon CloudWatch Observability
```
